src/patternfly/components/InputGroup/input-group-icon.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-input-group__icon{{#if input-group-icon--modifier}} {{input-group-icon--modifier}}{{/if}}" 
+  {{#if input-group-icon--attribute}}
+    {{{input-group-icon--attribute}}}
+  {{/if}}>
+  <i class="fas{{#if input-group-icon--type}} fa-{{input-group-icon--type}}{{/if}}" aria-hidden="true"></i>
```

Comment:
if `input-group-icon--type` isn't passed, it will render a broken icon, so we could either just remove the conditional since it's required, or you could do something like 

```
{{#if input-group-icon--type}}
  <i class="fas fa-{{input-group-icon--type}}" aria-hidden="true"></i>
{{else}}
  {{> @partial-block}}
{{/if}}